spharpy/plot/spatial.py

```diff
@@ -88,16 +87,16 @@ def _triangulation_sphere(sampling, data):
     triangulation : matplotlib Triangulation
 
     """ # noqa: 501
-    sampling = convert_coordinates(sampling)
     x, y, z = sph2cart(
+        sampling.azimuth,
+        sampling.colatitude,
         np.abs(data),
-        sampling.elevation,
```

Comment:
I don't know the previous implementation. If elevation was between 0 and pi its fine. Just asking because in pyfar colatitude is between 0 and pi and elevation between -pi/2 and pi/2